sklearn/preprocessing/data.py

```diff
@@ -409,6 +412,11 @@ def minmax_scale(X, feature_range=(0, 1), axis=0, copy=True):
     copy : boolean, optional, default is True
         Set to False to perform inplace scaling and avoid a copy (if the input
         is already a numpy array).
+
+    See also
+    --------
+    MinMaxScaler: Performs scaling to a given range using the``Transformer`` API
+         (e.g. as part of a preprocessing :class:`sklearn.pipeline.Pipeline`)
```

Comment:
Same as [this comment](https://github.com/scikit-learn/scikit-learn/pull/6748/files#r61788862)